tests/test_backend.py

```diff
@@ -208,534 +200,536 @@ def test_save_dict(self):
-        self.assertFalse(bool(res))
+        res = default_cache.delete("a")
+        assert not bool(res)
 
-    @patch("django_redis.cache.DJANGO_VERSION", (3, 1, 0, "final", 0))
-    def test_delete_return_value_type_new31(self):
+    def test_delete_return_value_type_new31(self, default_cache, mocker):
         """delete() returns a boolean instead of int since django version 3.1"""
-        self.cache.set("a", 1)
-        res = self.cache.delete("a")
-        self.assertEqual(type(res), bool)
-        self.assertTrue(res)
-        res = self.cache.delete("b")
-        self.assertEqual(type(res), bool)
-        self.assertFalse(res)
-
-    @patch("django_redis.cache.DJANGO_VERSION", (3, 0, 1, "final", 0))
-    def test_delete_return_value_type_before31(self):
+        mocker.patch("django_redis.cache.DJANGO_VERSION", new=(3, 1, 0, "final", 0))
+        default_cache.set("a", 1)
+        res = default_cache.delete("a")
+        assert type(res) == bool
+        assert res
+        res = default_cache.delete("b")
+        assert type(res) == bool
+        assert not res
+
+    def test_delete_return_value_type_before31(self, default_cache, mocker):
         """delete() returns a int before django version 3.1"""
-        self.cache.set("a", 1)
-        res = self.cache.delete("a")
-        self.assertEqual(type(res), int)
-        self.assertEqual(res, 1)
-        res = self.cache.delete("b")
-        self.assertEqual(type(res), int)
-        self.assertEqual(res, 0)
-
-    def test_delete_many(self):
-        self.cache.set_many({"a": 1, "b": 2, "c": 3})
-        res = self.cache.delete_many(["a", "b"])
-        self.assertTrue(bool(res))
-
-        res = self.cache.get_many(["a", "b", "c"])
-        self.assertEqual(res, {"c": 3})
-
-        res = self.cache.delete_many(["a", "b"])
-        self.assertFalse(bool(res))
-
-    def test_delete_many_generator(self):
-        self.cache.set_many({"a": 1, "b": 2, "c": 3})
-        res = self.cache.delete_many(key for key in ["a", "b"])
-        self.assertTrue(bool(res))
-
-        res = self.cache.get_many(["a", "b", "c"])
-        self.assertEqual(res, {"c": 3})
-
-        res = self.cache.delete_many(["a", "b"])
-        self.assertFalse(bool(res))
-
-    def test_delete_many_empty_generator(self):
-        res = self.cache.delete_many(key for key in [])
-        self.assertFalse(bool(res))
-
-    def test_incr(self):
-        if isinstance(self.cache.client, herd.HerdClient):
+        mocker.patch("django_redis.cache.DJANGO_VERSION", new=(3, 0, 1, "final", 0))
+        default_cache.set("a", 1)
+        res = default_cache.delete("a")
+        assert type(res) == int
+        assert res == 1
+        res = default_cache.delete("b")
+        assert type(res) == int
+        assert res == 0
+
+    def test_delete_many(self, default_cache):
+        default_cache.set_many({"a": 1, "b": 2, "c": 3})
+        res = default_cache.delete_many(["a", "b"])
+        assert bool(res)
```

Comment:
```suggestion
        assert res is True
```

Reply:
Deletion of the multiple keys returns back the number of deleted objects and therefore I used ` assert bool(res) is True` to cover these cases.